src/commands/compile/test_compile.py

```diff
@@ -0,0 +1,54 @@
+import json
+import shutil
+from pathlib import Path
+
+import pytest
+
+from src.commands.compile.compile import compile_contract
+
+
+current_directory = Path(__file__).parent
+mock_outputs = Path(current_directory, "mock_output")
+
+
+@pytest.fixture(name="clear_mock_outputs")
+def clear_mock_outputs():
+    if mock_outputs.is_dir():
+        shutil.rmtree(str(mock_outputs))
+    mock_outputs.mkdir()
```

Comment:
Output files can be stored in `tmpdir`, which is automatically cleared after each test.